src/views/partials/adminSidebar.hbs

```diff
@@ -0,0 +1,43 @@
+  <section class="sidebar">
+    <ul>
+      <a href="/admin">
+        <li>
+          <img class="icon" src="/img/web-page-home.png">
+          <span class="menu">HOME</span>
+        </li>
+      </a>
+      <a href="/admin/cohorts">
+        <li>
+          <img class="icon" src="/img/multiple-users-silhouette.png">
+          <span class="menu">Cohorts</span>
+        </li>
+      </a>
+
+      <a>
+        <li onclick="subMenu()">
```

Comment:
Fix indentation, and while you're here rename `subMenu` function to something more descriptive to the functionality.

Reply:
it's for a sub menu, why change it ?